data/ueber.json

```
@@ -1,22 +1,22 @@
     },
     {
         "title": "Politik",
-        "txt": "Für Entscheidungsträger&ast;innen in der Politik sind Open Government Strategien von besonderer Bedeutung. Denn durch diese wird der Dialog zwischen Bürger&ast;innen und Politiker&ast;innen verstetigt und öffentliches Wissen auf eine breite Basis gestellt. Im Austausch mit der Code for Germany Community können sie mehr über die Probleme und digitalen Lösungsansätze in ihren Wahlkreisen erfahren und ihre Verantwortung gegenüber den Wähler*innen stärken."
+        "txt": "Entscheidungsträger\*innen in der Politik können sich in Gesetzgebungsprozessen für Transparenz, Offene Daten und Open-Government-Strategien einsetzen. Damit stärken sie demokratische Verfahren, Wissensbildung und Partizipation. Im Austausch mit unserer Community erfahren sie mehr über die digitalen Lösungsansätze für Herausforderungen in ihren Wahlkreisen und können den Austausch zwischen sich und der Zivilgesellschaft entsprechend suchen und fördern."
     },
     {
         "title": "Unternehmen",
-        "txt": "Jedes Unternehmen das  öffentliche Datenbestände hat, kann durch deren Bereitstellung für Civic Tech eine Menge über seine eigene Zielgruppe lernen. Sie fördern damit Innovationen, die der Allgemeinheit zugute kommen. Darüber hinaus setzen sie Datenstandards, die wiederum als Vorbild für Politik und Verwaltung dienen können."
+        "txt": "Jedes Unternehmen kann durch die Bereitstellung öffentlicher Daten Innovationen fördern, die der Allgemeinheit nützen. Damit ermöglichen sie auch neue Geschäftsmodelle. Sie beteiligen sich zudem an der Schaffung und Durchsetzung von Datenstandards."
     },
     {
         "title": "Organisationen und Förderer",
-        "txt": "Mit dem deutschlandweiten Netzwerk kann Code for Germany geteilte Ziele wie gesellschaftlichen Zusammenhalt und das Verständnis für die Möglichkeiten digitaler Anwendungen zu fördern, erreichen. Der Stellenwert ehrenamtlicher Projekte wird in den Labs hervorgehoben. Auch vernachlässigte Problemfelder erhalten durch neue digitale Lösungen mehr Aufmerksamkeit und können damit neue Zielgruppen erschließen."
+        "txt": "Code for Germany kann mit seinen Partner\*innen gemeinsame Ziele wie gesellschaftlichen Zusammenhalt sowie den bedürfnisorientierten Einsatz digitaler Anwendungen erreichen. Mit Open Government erhalten auch vernachlässigte Problemfelder in der Interaktion zwischen Verwaltungen und Bürger\*innen mehr Aufmerksamkeit. Dafür muss die Befreiung von Daten unterstützt werden."
+    }
```

Hier fehlt ein Komma
```suggestion
    },
```